### PR TITLE
Change affiliation logic in author profile

### PR DESCRIPTION
**Pull request by NohYujeong**

### CODE REVIEW

File: app/components/layouts/components/suggestionList.scss [hunk 1]
```diff
@@ -39,7 +39,40 @@
...
   }
 
-  b {
-    color: $main_blue1;
+  .keywordCompletionItemContext {
+    font-size: 13px;
+    color: $gray6;
+
+    b {
+      color: #656d7d;
```
Comment (TylorShin) [inline app/components/layouts/components/suggestionList.scss]: is there any color variable?

File: app/components/layouts/components/suggestionList.tsx [hunk 2]
```diff
@@ -1,6 +1,8 @@
 import * as React from "react";
 import { escapeRegExp } from "lodash";
 import { withStyles } from "../../../helpers/withStylesHelper";
+import Icon from "../../../icons";
+import * as classNames from "classnames";
```
Comment (TylorShin) [inline app/components/layouts/components/suggestionList.tsx]: Please change the position of the npm module import

File: app/components/layouts/components/suggestionList.tsx [hunk 3]
```diff
@@ -79,29 +81,43 @@ const SuggestionList: React.SFC<SuggestionListProps> = props => {
         e.preventDefault();
         props.handleClickSuggestionKeyword(props.suggestionList[index]);
       }}
-      className={styles.keywordCompletionItem}
+      className={classNames({
+        [`${styles.keywordCompletionItem}`]: index != 0,
```
Comment (TylorShin) [inline app/components/layouts/components/suggestionList.tsx]: `[styles.keywordCompletionItem]` will be fine.

File: app/components/layouts/components/suggestionList.tsx [hunk 4]
```diff
@@ -79,29 +81,43 @@ const SuggestionList: React.SFC<SuggestionListProps> = props => {
...
+        [`${styles.highLightKeywordCompletionItem}`]: index === 0,
```
Comment (TylorShin) [inline app/components/layouts/components/suggestionList.tsx]: same here.

File: app/components/layouts/components/suggestionList.tsx [hunk 5]
```diff
@@ -79,29 +81,43 @@ const SuggestionList: React.SFC<SuggestionListProps> = props => {
...
       }}
       key={`keyword_completion_${suggestion}${index}`}
       tabIndex={-1}
     >
-      <span dangerouslySetInnerHTML={{ __html: suggestion }} />
+      <span className={styles.keywordCompletionItemContext} dangerouslySetInnerHTML={{ __html: suggestion }} />
     </a>
   ));
-
+  highlightedList.length;
```
Comment (TylorShin) [inline app/components/layouts/components/suggestionList.tsx]: What is the purpose of this line?

File: app/components/layouts/components/suggestionList.tsx [hunk 6]
```diff
@@ -79,29 +81,43 @@ const SuggestionList: React.SFC<SuggestionListProps> = props => {
...
   return (
     <div style={{ display: props.isOpen ? "block" : "none" }} className={styles.keywordCompletionWrapper}>
       {highlightedContent}
+      {props.userInput.length > 0 ? (
```
Comment (TylorShin) [inline app/components/layouts/components/suggestionList.tsx]: In this case, I personally prefer to use `props.userInput.length > 0 &&` syntax.

File: app/components/layouts/components/suggestionList.tsx [hunk 7]
```diff
@@ -79,29 +81,43 @@ const SuggestionList: React.SFC<SuggestionListProps> = props => {
...
+        <a
+          onMouseDown={e => {
+            e.preventDefault();
+            props.handleClickSuggestionKeyword(props.userInput);
+          }}
+          className={classNames({
+            [`${styles.keywordCompletionItem}`]: highlightedList.length > 0,
```
Comment (TylorShin) [inline app/components/layouts/components/suggestionList.tsx]: Don't need to wrap.

File: app/components/layouts/components/suggestionList.tsx [hunk 8]
```diff
@@ -79,29 +81,43 @@ const SuggestionList: React.SFC<SuggestionListProps> = props => {
...
+            [`${styles.highLightKeywordCompletionItem}`]: highlightedList.length === 0,
```
Comment (TylorShin) [inline app/components/layouts/components/suggestionList.tsx]: don't need to wrap here too.

File: app/components/layouts/components/suggestionList.tsx [hunk 9]
```diff
@@ -79,29 +81,43 @@ const SuggestionList: React.SFC<SuggestionListProps> = props => {
...
+      ) : (
+        ""
+      )}
     </div>
   );
 };
 
 export default withStyles<typeof SuggestionList>(styles)(SuggestionList);
-
-// const targetSearchQueryParams = PapersQueryFormatter.stringifyPapersQuery({
```
Comment (TylorShin) [inline app/components/layouts/components/suggestionList.tsx]: 👍

File: app/components/layouts/components/suggestionList.scss (hunk 1 above)
Comment (NohYujeong) [reply app/components/layouts/components/suggestionList.scss]: Yes

File: app/components/layouts/components/suggestionList.tsx (hunk 5 above)
Comment (NohYujeong) [reply app/components/layouts/components/suggestionList.tsx]: That's my testing trace. My mistake! I will remove.

File: app/components/layouts/components/suggestionList.scss (hunk 1 above)
Comment (TylorShin) [reply app/components/layouts/components/suggestionList.scss]: ...I think of that you mean 'no'

**Comment by TylorShin** [comment]
Please resolve conflicts

**Comment by NohYujeong** [comment]
> Please resolve the conflicts.

OK! I got it

**Pull request closed without merging**